CoseSign1.Certificates/Local/Validators/X509ChainTrustValidator.cs

```diff
@@ -143,7 +165,7 @@ protected override CoseSign1ValidationResult ValidateCertificate(
             }
         }
 
-        // If we're here, chain build failed.
+        // If we're here, chain build failed. We need to filter out the errors we're willing to ignore.

```

Comment:
For the "user-supplied root" section - could we make that only compile in netstandard?

Reply:
Yeah that's a good idea. I think I will leave that for another update as it would require a bit of reworking since it is actually used in some net8 flows - specifically when TrustUserRoots == false